saleor/graphql/core/types/common.py

```diff
@@ -448,7 +448,8 @@ def resolve_url(root, info):
         # check if URL is absolute:
         if urlparse(root.url).netloc:
             return root.url
-        return build_absolute_uri(default_storage.url(root.url))
+        # unquote used for preventing double URL encoding
+        return build_absolute_uri(default_storage.url(unquote(root.url)))
```

Comment:
Why are we saving quoted names in the database if they are not quoted in the file system?